common/src/types/v0/transport/cluster_agent.rs

```diff
@@ -83,6 +83,42 @@ impl ReportFailedPaths {
     }
 }
 
+#[derive(Debug)]
+pub struct GetController {
+    nvme_path: String,
+}
+
+impl GetController {
+    pub fn new(nvme_path: String) -> Self {
+        Self { nvme_path }
+    }
+    pub fn nvme_path(&self) -> String {
+        self.nvme_path.clone()
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct NvmeSubsystem {
+    address: String,
+}
+
+impl NvmeSubsystem {
+    pub fn new(target_addr: String) -> Self {
+        Self {
+            address: target_addr,
+        }
+    }
+    pub fn address(&self) -> String {
+        self.address.clone()
+    }
+}
+
+// impl From<NvmeSubsystem> for String {
```

Comment:
Is this commented block needed? If not, let's delete the same.

Reply:
Removed it now.